packages/shared/lib/services/sync/job.service.ts

```diff
@@ -148,10 +148,10 @@ export const isSyncJobRunning = async (sync_id: string): Promise<Pick<SyncJob, '
             deleted: false,
             status: SyncStatus.RUNNING
         })
-        .first('id', 'job_id', 'run_id');
+        .orderBy('created_at', 'desc');
```

Comment:
Should we keep the limit?

```suggestion
        .orderBy('created_at', 'desc').limit(1);
```

Reply:
Updated, thanks